.travis.yml

```diff
@@ -39,18 +40,18 @@ install:
   - if [ $DEP_MANAGEMENT_TOOL == "DEP" ]; then
       dep ensure;
     else
-      go get;
+      go get all;
```

Comment:
Apologies for the conflicting requests, but could you actually change this to `go mod download`? This command installs the latest version of some dependencies which causes our tests to break. We run `go mod download` for GitHub actions which does not run into the same issues.